test/fx2trt/trt_lower/test_diagnostics.py

```diff
@@ -0,0 +1,184 @@
+# Owner(s): ["oncall: aiacc"]
```

Comment:
is this file supposed to be added?

Reply:
Sorry this is the file that this diff was rebased on. Should be gone already.